xm-commons-logging-trace/src/main/java/com/icthh/xm/commons/logging/trace/SleuthWrapperOld.java

```diff
@@ -0,0 +1,64 @@
+//package com.icthh.xm.commons.logging.trace;
```

Comment:
Given that the SleuthWrapperOld class is newly created and is entirely commented out, it raises some questions about its purpose in the project. If there are no immediate plans for this class, it would be advisable to remove it to keep the codebase clean